src/pages/home/home.ts

```diff
@@ -212,6 +213,7 @@ export class HomePage {
 
     this.homeIntegrations = _.remove(integrations, x => {
       this.showBuyCryptoOption = x.name == 'simplex' && x.show == true;
+      this.showShoppingOption = x.name == 'giftcards' && x.show == true;
```

Comment:
This function is iterative, so the value could be false if the latest element is not `giftcards`